.changeset/eleven-ears-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
```

Comment:
thinking out loud: Is this really just a patch? potentially is a visual change in the places where it's used. maybe we should use a "minor" bump here?

Reply:
@Dhaulagiri thoughts on this? sorry to drag in most versioning conversations, but I think we'll need a bit of guidance until we (or at least I) get to a common sense on these 😁

Reply:
> sorry to drag you in most versioning conversations

Not a problem as I feel like I'm often the ones starting them 😅 

Even though my experience is that there is an inherent subjectivity to these choices which makes it hard to create "rules", it does strike me that even a light framework we could reference might be useful. I'll add a task to our backlog.